src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -2411,6 +2411,7 @@ def test_aks_create_aadv1_and_update_with_managed_aad(self, resource_group, reso
                      '--aad-server-app-secret fake-secret ' \
                      '--aad-client-app-id 00000000-0000-0000-0000-000000000002 ' \
                      '--aad-tenant-id d5b55040-0c14-48cc-a028-91457fc190d9 ' \
+                     '--aks-custom-headers AKSHTTPCustomFeatures=Microsoft.ContainerService/AADv1AllowCreate' \
```

Comment:
Queued [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=78469847&view=results) to validate the change

Reply:
Queued a new [one](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=78469928&view=results).